library/card/card.css

```diff
@@ -88,70 +88,70 @@
                          'cardActions cardActions cardActions cardActions cardActions cardActions';
 }
 
-.card__img--basic-with-dismiss-button {
+.dui-card__img--basic-with-dismiss-button {
     object-fit: cover;
     width: 100%;
     height: 100%;
 }
 
-.card__actions--basic-with-dismiss-button  {
+.dui-card__actions--basic-with-dismiss-button  {
```

Comment:
class names are too long

Reply:
Yup will be consolidating in further iterations.